backend/routers/admin_products.py

```
@@ -235,6 +235,68 @@ async def update_pricing_config(config_key: str, config: dict = Body(...), db=De
         return {"success": False, "error": "Config not found"}, 404
     return {"success": True}
 
+@router.get("/pricing/welcome-credits")
+async def get_welcome_credits_config(db=Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)